<ui version="4.0" >
 <class>ExportSecretKeyDialog</class>
 <widget class="QDialog" name="ExportSecretKeyDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Export Secret Key</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="descriptionLB" >
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="outputFileLB" >
     <property name="text" >
      <string>Output file:</string>
     </property>
     <property name="buddy" >
      <cstring>outputFileFR</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="charsetLB" >
     <property name="text" >
      <string>Passphrase charset:</string>
     </property>
     <property name="buddy" >
      <cstring>charsetCB</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QComboBox" name="charsetCB" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QCheckBox" name="armorCB" >
     <property name="text" >
      <string>ASCII armor</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <widget class="Kleo::FileNameRequester" native="1" name="outputFileFR" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Kleo::FileNameRequester</class>
   <extends>QWidget</extends>
   <header>ui/filenamerequester.h</header>
   <container>1</container>
   <slots>
    <signal>fileNameChanged(QString)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportSecretKeyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>227</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportSecretKeyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>295</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>armorCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>ExportSecretKeyDialog</receiver>
   <slot>updateFileName()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>77</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel" >
     <x>69</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outputFileFR</sender>
   <signal>fileNameChanged(QString)</signal>
   <receiver>ExportSecretKeyDialog</receiver>
   <slot>updateFileName()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>267</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel" >
     <x>159</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateFileName()</slot>
 </slots>
</ui>
